README.md

```diff
@@ -67,6 +67,31 @@ Note that testing is done on specific minor versions, but any minor versions wit
   - Self-managed (on-premises)
   - Managed cloud offerings (e.g., Amazon OpenSearch, Amazon OpenSearch Serverless)
 
+### Performance Limitations
+A performance test was performed on 03/10/25 alongside [PR 1337](https://github.com/opensearch-project/opensearch-migrations/pull/1337)
+to identify general indexing throughput with Reindex-From-Snapshot and Traffic Replay. While Reindex-From-Snapshot
+includes periods of ingestion inactivity (e.g. while downloading the shard data), this was not factored into the ingestion rates.
+Test used docs with an average uncompressed size 2.39 KB (source doc and index command) which corresponded to 1.54 KB of primary shard size per doc on OS 2.17 with default settings
+
+For real-word use, throughput can be increased by vertically scaling Reindex-From-Snapshot and Traffic Replay instances or horizontally scaling
```

Comment:
Is there any test that potentially tests the vertical scaling max? The throughput with 2vCPU is nice to see but it would also potentially be nice to see the actual "max" performance say at 16vCPU or even just 4vCPU for a rough comparison.

(Sorry for the random comments, I can create an issue if that's a better way to track more detailed performance tests)

Reply:
Thanks for taking the time to follow up! 

This does make sense and roughly what I expected. I think since this was published on the repo's main README I initially read as a performance limitation of the solution and and forgot about worker  per shard parallelization mentioned here: https://github.com/opensearch-project/opensearch-migrations/blob/main/RFS/docs/DESIGN.md#how-the-rfs-worker-works